src/js/slider/slider.js

```diff
@@ -353,7 +327,7 @@ class Slider extends Component {
    *        Event that caused this object to run
    */
   handleClick(event) {
-    event.stopImmediatePropagation();
```

Comment:
stopImmediatePropagation calls stopPropagation, though, not sure why immediate is used here.

Reply:
Yeah, it seemed pointless since we turn off the capture phase everywhere.